Modules/Transformation/src/BSplineFreeFormTransformationStatistical.cc

```diff
@@ -42,8 +42,8 @@ ::BSplineFreeFormTransformationStatistical()
 BSplineFreeFormTransformationStatistical
 ::BSplineFreeFormTransformationStatistical(const ImageAttributes    &attr,
                                            CPStatus              ****status,
-                                           const Matrix             &mat,
-                                           const Vector             &vec)
+                                           const mirtk::Matrix      &mat,
```

Comment:
> An SDM file is not and FFD nor a DOF.

That's what I think is maybe not the best way to do it. A SDM could be considered a Statistical FFD. One without coefficient vector because this vector is all zero (i.e., a sparse representation of this vector which does not take up any space). The input to a SDM based registration would be this transformation file as ```-dofin``` and the output ```-dofout``` is one with a non-zero coefficient vector.